changelog.yaml

```diff
@@ -2975,3 +2975,8 @@
     changed:
     - CalEITC reimplemented without branching and without bug.
   date: 2023-04-14 10:05:46
+- bump: patch
```

Comment:
This content goes in changelog_entry.yaml, not changelog.yaml. You can see other recently merged PRs for examples.